entries/a/app.ramp.com.json

```
@@ -0,0 +1,12 @@
+{
+  "Ramp": {
+    "domain": "app.ramp.com",
+    "additional-domains": ["ramp.com", "demo.ramp.com"],
```

The domain should be without a subdomain.
```suggestion
    "domain": "ramp.com",
```

---

The domains with actual login screens which use MFA are `app.ramp.com` and `demo.ramp.com`. Is it still appropriate to use just the top level domain? Will adjust (and rename the file) if so!

---

Yes we don't list subdomains. Third party apps using our data will look for the top level domain and not the full domain.